students/DrewSmith/lessons/lesson03/assignment/tests/test_basic_operations.py

```diff
@@ -0,0 +1,132 @@
+'''
+Tests the basic_operations module
+'''
+
+
+from pytest import fixture
+from pytest import raises
+import datetime
+import assignment.create_customers as cc
+# Use Test environment database
+cc.model_setup(environment = "test")
+
+import assignment.basic_operations as bo
+from assignment.customers_model import Customer
+
+
+customer_id_list = ("search_customer", "delete_customer_good", "update_credit", "active_customers")
+
+customer_data = { 
+    'customer_id': '',
+    'name': 'Suzie',
+    'lastname': "Edgar",
+    'home_address': "123 Browncroft Blv, Rochester, NY 97235",
+    'phone_number': '123-456-7890',
+    'email_address': 'suzie@hotmail.com',
+    'status': True,
+    'credit_limit': '40'
+}
+
+@fixture(autouse=True)
+def setup_teardown():
+    ''' Fixture to execute before and after tests '''
+    cc.DATABASE.drop_tables([Customer])
+    cc.DATABASE.create_tables([Customer])
+    
+    for customer_id in customer_id_list:
+        copy = dict(customer_data)
+        copy["customer_id"] = customer_id
+        bo.add_customer(**copy)
+
+    yield
+
+    cc.DATABASE.drop_tables([Customer])
```

Comment:
Great fixture. You actually don't need the drop at the end, since you have it at the beginning.
I don't think you need to copy your dictionary. You aren't using it for anything, just send it in, as is.